tensorflow/compiler/mlir/hlo/runtime/BUILD

```diff
@@ -0,0 +1,89 @@
+load("//tensorflow/core/platform:rules_cc.bzl", "cc_library")
+
+# TODO(b/160617323): Decouple MLIR HLO from TensorFlow/XLA
+load("//tensorflow:tensorflow.bzl", "tf_cc_test")
+
+package(
+    default_visibility = [":friends"],
+    licenses = ["notice"],
+)
+
+package_group(
+    name = "friends",
+    includes = ["//third_party/mlir:subpackages"],
+    packages = [
+        "//babelfish/device/...",
+        "//learning/brain/experimental/dtensor/...",
+        "//learning/brain/experimental/mlir/...",
+        "//learning/brain/google/xla/kernels/...",
+        "//learning/brain/google/xla/mlir/...",
+        "//learning/deepmind/partir/...",
+        "//learning/pathways/data_parallel/tf2xla/...",
+        "//platforms/xla/...",
+        "//third_party/iree/...",
+        "//third_party/mlir_edge/...",
+        "//third_party/py/mlir/...",
+        "//tensorflow/compiler/mlir/...",
+        "//tensorflow/compiler/tf2xla/...",
+        "//tensorflow/compiler/xla/...",
+        # Allow visibility from the mlir language server.
+        "//learning/brain/mlir/mlir_lsp_server/...",
+    ],
```

Comment:
That's a lot of visibility, can you cut this down?

Reply:
Done!